docker/run_bot_task.py

```diff
@@ -503,30 +501,22 @@ def parse_meta_yaml(
     "--platform",
     type=str,
     default=None,
-    help="The platform (e.g., 'linux', 'osx', 'win').",
+    help="The platform (e.g., 'linux-64', 'osx-arm64', 'win-64').",
```

Comment:
Just an FYI in conda platform is linux, osx, win and arch is x86_64 (aka 64), arm64, ppc64le, etc. IDK what rattler-build is doing, but the doc string and usage here is confusing. I'd rename this option to `--platform-arch` if that is how it is supposed to be used.

Reply:
We should also double check the code to ensure the correct value is being passed here.